src/Utilities/Idm/SourceLoad.F90

```diff
@@ -168,6 +171,13 @@ subroutine load_simnam()
       !
       ! -- open namfile and load to input context
       call input_load(simfile, mf6_input, simfile, iout)
+      !
+      ! -- load optional HPC configuration file
+      if (filein_fname(hpc6_filename, 'HPC6_FILENAME', mf6_input%mempath, &
+                       simfile)) then
+        hpc_input = getModflowInput('HPC6', 'UTL', 'HPC', 'UTL', 'HPC', simfile)
+        call input_load(hpc6_filename, hpc_input, hpc6_filename, iout)
```

Comment:
In this line the 3rd parameter "hpc6_filname" (second occurrence) should instead be "simfile".  This second filename argument is used to set an error associated with opening the file as that is where the filename originated from.